### DISCUSSION TITLE
XHTML Extension Support

### DISCUSSION THREAD

**Discussion by ScottDutton3** [phcode-dev/phoenix#974] [Q&A] [answered]
Good to see Brackets still alive with Phoenix.

I use it primarily for ebook creation. I could use xhtml extension support. As it is, I have to change one file to html before it will preview, then turn it back to xhtml so I can preview all files with that extension. Eventually, Phoenix forgets that it CAN preview those files and I have to do the small hack again.

Thanks very much.

**Comment by abose** [comment]
@ScottDutton3 Thanks for the feature request. Can you attach a sample XHTML project for me to test the workflow. And any instructions on how the workflow should be on the project ideally.

**Comment by ScottDutton3** [comment]
Hi Arun -

Sure, I've attached an archive of a completed ebook project. Change the .zip extension to .epub and you could load it into an ereader.

The book content is set up as XHTML files. Other files are XML, including content.opf and toc.ncx.

Phoenix recognises the .css and .xml files correctly. But the .xhtml extension needs to be changed to .html before it will preview.

Thanks for your help.

- Scott

...................................................
Design + Illustration
https://www.catspawdynamics.com





> On Feb 15, 2023, at 9:47 PM, Arun Bose ***@***.***> wrote:
> 
> 
> @ScottDutton3 <https://github.com/ScottDutton3> Thanks for the feature request. Can you attach a sample XHTML project for me to test the workflow. And any instructions on how the workflow should be on the project ideally.
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/phcode-dev/phoenix/discussions/974#discussioncomment-4990499>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/A55PCW74OYERYL4NBF7522DWXWWMNANCNFSM6AAAAAAU5F746M>.
> You are receiving this because you were mentioned.
> 

￼

**Reply by abose** [reply]
Hi @ScottDutton3, I can't download the zip file from the discussion. Github might only allow drag and drop the zip file directly in the discussion comment. 

But I was able to see the issue with a simple xhtml file. I will fix it soon. But in the meantime, can you upload the zip file to verify?

**Reply by abose** [reply]
Track fix here: https://github.com/phcode-dev/phoenix/issues/981

**Comment by abose** [comment, marked as answer]
XHTML files should now be able to be live previewed in https://phcode.dev . The update is being rolled out and you should be able to use this feature after a few reloads of phcode in browser.

**Comment by ScottDutton3** [comment]
Hi Arun -

Sorry for the delay getting back to you. Yes, XHTML files now preview correctly.

I appreciate your help.

- Scott

---
